atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/ValidatingQueryRewritingKeyValueService.java

```diff
@@ -103,7 +103,7 @@ protected static void sanityCheckTableName(TableReference tableRef) {
                 "invalid tableName: " + tableName);
     }
 
-    protected static void sanityCheckTableMetadata(TableReference tableRef, byte[] tableMetadata) {
+    protected void sanityCheckTableMetadata(TableReference tableRef, byte[] tableMetadata) {
```

Comment:
does this one actually need to be non-static? looks like it was added in clockfort's PR